app/javascript/gobierto_dashboards/webapp/lib/utils.js

```diff
@@ -11,3 +11,12 @@ export function getRemoteData(endpoint) {
       });
   })
 }
+
+export function uuid() {
+  return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, (c) => {
```

Comment:
Holy s***! Are the *4, y* common uuid values? What is the purpose of this function? Is specific? Add some comments to clarify weird stuff

Reply:
Uhm, I thought I needed this, but I realised I don't while I was writing to you explaining why I used it. I'll remove it and explain. Thanks.